src/index.js

```diff
@@ -96,9 +96,15 @@ module.exports = async function App(config) {
   // Send all individual attachments to their respective channels per team.
   if (subChannelAttachments.length) {
     await Promise.all(
-      subChannelAttachments.map(({ attachment, channel: subChannel }) =>
-        releaseCommunication.sendMessage(message, [attachment], subChannel, true),
-      ),
+      subChannelAttachments.map(async ({ attachment, channel: subChannel }) => {
+        try {
+          releaseCommunication.sendMessage(message, [attachment], subChannel, true);
```

Comment:
Do we want to `await` here still so that `Promise.all` isn't resolved until each `sendMessage` is settled?

```suggestion
          await releaseCommunication.sendMessage(message, [attachment], subChannel, true);
```

Reply:
Unless the intent is to let these happen non-blocking in the background now?

Reply:
I went back and forth on that.  Seems more like a non-blocking fire and forget without trying to impact the other calls.  Nothing is done with the responses unless an error occurs and we don't want the error to cause other calls to fail.  Thoughts? I am good either way

Reply:
Might be the only way to know when captains log is done though so going to add it

Reply:
@JoshHarris Good points. If we want to wait for complete but not fail everything if one fails, we could try `Promise.allSettled`.